ArduCopter/mode_autotune.cpp

```diff
@@ -180,16 +180,6 @@ bool Copter::ModeAutoTune::start(bool ignore_checks)
         return false;
     }
 
-    // initialize vertical speeds and leash lengths
-    pos_control->set_speed_z(-get_pilot_speed_dn(), g.pilot_speed_up);
-    pos_control->set_accel_z(g.pilot_accel_z);
-
-    // initialise position and desired velocity
-    if (!pos_control->is_active_z()) {
-        pos_control->set_alt_target_to_current_alt();
-        pos_control->set_desired_velocity_z(inertial_nav.get_velocity_z());
-    }
-
     return true;
```

Comment:
Shouldn't this call base implementation?

Reply:
Yes, fixed.